### PR TITLE
Internet access from mac address

### PR DESCRIPTION
**Pull request by leonardlan**
Issue #198 
Adds page to grant internet access to devices from mac address @radicalray

### CODE REVIEW

File: ap/web_access/views.py [hunk 1]
```diff
@@ -55,24 +56,29 @@ def get_queryset(self):
             return WebRequest.objects.filter().order_by('status')
 
 
-class TAWebRequestList(generic.ListView):
+class TAWebRequestList(GroupRequiredMixin, generic.ListView):
 
     model = WebRequest
     template_name = 'web_access/ta_webrequest_list.html'
     context_object_name = 'web_access'
+    group_required = ['administration']
```
Comment (radicalray) [inline ap/web_access/views.py]: change that to ```networks```, administration is like office sisters.

File: ap/web_access/views.py [hunk 2]
```diff
@@ -55,24 +56,29 @@ def get_queryset(self):
...
 
 
-class TAWebAccessUpdate(generic.UpdateView):
+class TAWebAccessUpdate(GroupRequiredMixin, generic.UpdateView):
 
     model = WebRequest
     template_name = 'web_access/ta_web_access_update.html'
     form_class = WebAccessRequestTACommentForm
     context_object_name = 'web_access'
+    group_required = ['administration']
```
Comment (radicalray) [inline ap/web_access/views.py]: Same here to: ```ta```

File: ap/web_access/views.py [hunk 3]
```diff
@@ -120,3 +126,20 @@ def createGuestWebAccess(request):
 def deleteGuestWebAccess(request, id):
     WebRequest.objects.filter(id=id).delete()
     return getGuestRequests(request)
+
+
+@group_required(('administration', 'networks'), raise_exception=True)
```
Comment (radicalray) [inline ap/web_access/views.py]: here too.

**Comment by radicalray** [comment]
@leonardlan Looks generally good. Fix the minor issues and let's get this merged. Good job!

File: ap/web_access/views.py (hunk 1 above)
Comment (leonardlan) [inline ap/web_access/views.py]: I don't see "ta" in ap/aputils/groups.py file. Are these where the permission groups are defined?

	permission_list = [
		'administration',
		'maintenance',
		'absent_trainee_roster',
		'attendance_monitors',
		'av',
		'dev',
		'networks',
		'exam_graders',
		'grad_committee',
		'HC',
		'facility_maintenance_or_frames_or_linens',
		'house_inspectors',
		'semi_annual_testing_group_coordinators',
		'service_schedulers',
		'team_monitors',
		'ypc_monitors',
		'xb_trainees',
		'designated_service',
		'special_projects',
		'office_support',
		'badges',
		'health_office',
		'kitchen',
	]

**Comment by radicalray** [comment]
Looks good. Good job, @leonardlan !

**Pull request merged**